lib/patches/bundler/version.rb

```
@@ -0,0 +1 @@
+raise 'unsupported bundler version please use 1.14.x' unless Bundler::VERSION =~ /^1\.14\./
```

Thanks. Silently failing created issues for me previously.